### DISCUSSION TITLE
Is it possible to use plotters to create a graph dynamically generated in a web page (<img>)?

### DISCUSSION THREAD

**Discussion by FritsHoogland** [plotters-rs/plotters#523] [Q&A]
The idea is to have a tool gather statistics, and serve a web page which can produce graphs/plots dynamically.
I’ve asked around already, most answers are to use WASM, but that beats the purpose of having a CLI tool with optional graphing option.

**Comment by gitmalong** [comment]
I can see two options in here
1) Rust WASM app for client side rendering
2) Rust backend that serves the image to your frontend
